content/en/docs/contribute/localization.md

```diff
@@ -239,37 +246,41 @@ Some language teams have their own language-specific style guide and glossary. F
 
 ## Branching strategy
 
-Because localization projects are highly collaborative efforts, we encourage teams to work in shared development branches.
+Because localization projects are highly collaborative efforts, we encourage teams to work in shared localization branches.
 
-To collaborate on a development branch:
+To collaborate on a localization branch:
```

Comment:
Hi @shu-mutou .
Is the main focus of this PR to change the branch terminology (development -> localization) in this guide?
I think it would be worthwhile to get some localization teams to review the changes.
@kubernetes/sig-docs-fr-reviews @kubernetes/sig-docs-ja-reviews @kubernetes/sig-docs-ko-reviews @kubernetes/sig-docs-zh-reviews 

Are you able to rebase your changes into a single commit? Thanks!

Reply:
> Is the main focus of this PR to change the branch terminology (development -> localization) in this guide?

No, the main purpose is the clarifications how l10n maintainers and translators should operate branches.